app/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "headlamp",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "copy-server-binary": "cd ../ && make backend && cp -r ./backend/server ./app/electron/",
+    "start-electron": "npm run copy-server-binary && cd ../ && make run-backend  & cd ../frontend/ && export BROWSER=none && npm start",
+    "build-electron": "npm run copy-server-binary && cd ../frontend/ && npm run build && cp -r ../app/electron/. build/electron",
+    "package-electron": "mkdir -p build && cp -r ../frontend/build/ ./build && electron-builder build --mac --win --linux -c.extraMetadata.main=build/electron/main.js --publish never",
```

Comment:
As mentioned on Slack, for this to work as intended, it should copy the contents of the `frontend/build`, not the whole folder as it will copy it into the existing `build` one, i.e. `cp -r ../frontend/build/* ./build`